HomeTask1.html

```diff
@@ -1,124 +1,124 @@
-          <button value="Continue Reading" class="button">Continue Reading</button>
-          </div>
-          <div id="three" style=" display: inline-block;">
-                  SELECT CATEGORY
-
-                  <select class="dropdown">
-                      <option value="All">ALL</option>
-                  </select>
-                  </br>
-                  <input type="text" name="fname" class="input">
-                  <button class="subscribebutton">Subscribe</button>
-
-
-        </div>
-</div>
-
-
-
-<hr>
-<br/>
-
-
-
-<div style="height:250px;width:90%;">
-
-
-
-<div style="width:15%;">
-<img src="image2.png" alt="image" class="image"/>
-</div>
-
-
-<div style="display:contents;float:left;">
-<h3 class="title">Title Two</h3>
-<br>Posted on 29th june // category:category one
-<p style="width:1300px;">We've helped over 120 Fortune 1000 companies in the following industries solve their most complex technology challenges. Now see what we can do for you.
-We've helped over 120 Fortune 1000 companies in the following industries solve their most complex technology challenges. Now see what we can do for you.
-We've helped over 120 Fortune 1000 companies in the following industries solve their most complex technology challenges. Now see what we can do for you.</p>
-<button value="Continue Reading" class="button">Continue Reading</button>
-</div>
-
-
-</div>
-
-
-
-
-
-
-<hr>
-<br/>
-
-
-<div style="height:250px;width:90%;">
-<img src="image2.png" alt="image" class="image"/>
-<div style="display:contents;float:left;width:300px;">
-<h3 class="title">Title Three</h3>
-<br>Posted on 29th june // category:category one
-<p style="width:1300px;">We've helped over 120 Fortune 1000 companies in the following industries solve their most complex technology challenges. Now see what we can do for you.
-We've helped over 120 Fortune 1000 companies in the following industries solve their most complex technology challenges. Now see what we can do for you.
-We've helped over 120 Fortune 1000 companies in the following industries solve their most complex technology challenges. Now see what we can do for you.</p>
-<button value="Continue Reading" class="button">Continue Reading</button>
-</div>
-</div>
-
-
-
-
-
-
-</div>
-
-
-
-<div class="footer">
-<h4 class="h4one">@NewsFeed 2019</h4>
-</div>
-
-
-
-
-
-
-</body>
-
-
-</html>
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+<link rel="stylesheet" href="css/HomeTask1.css">
+<title>
+FrontCamp Home Task one
+</title>
+<style>
+
+
+</style>
+</head>
+
+<body>
+
+
+  <div class="div1">
+  <h1 class="newsfeed" id="h1-one">NEWSFEED</h1>
+  <h4 class="h4one">Yet another newsfeed</h4>
+  </div>
+
+
+<div class="maindiv">
+
+
+
+
+<div id="outer" style=" display: inline-block;">
+        <div id="one" style=" display: inline-block;">
+          <img src="image1.png" alt="image" />
```

Comment:
Use relative paths here.
Alt attribute can be a meaningful one than just image.